x-pack/plugins/reporting/server/config/schema.test.ts

```diff
@@ -62,4 +62,23 @@ describe('Reporting Config Schema', () => {
       );
     }
   );
+
+  it('disables image reporting with serverless', () => {
+    // csv is permitted in all configs
+    expect(() =>
+      ConfigSchema.validate({ export_types: { csv: { enabled: true } } }, { serverless: true })
+    ).not.toThrow();
+  });
+
+  it('it should allow image reporting for any non-serverless config', () => {
+    expect(() =>
+      ConfigSchema.validate({ export_types: { pdf: { enabled: true } } }, { dev: true })
+    ).not.toThrow();
+    expect(() =>
+      ConfigSchema.validate({ export_types: { png: { enabled: true } } }, { dev: true })
+    ).not.toThrow();
+    expect(() =>
+      ConfigSchema.validate({ export_types: { csv: { enabled: true } } }, { dev: true })
+    ).not.toThrow();
+  });
```

Comment:
Let's add some tests to make sure the defaults are what we expect for the serverless environment:
```typescript
  it('enables all export types by default', () => {
    expect(ConfigSchema.validate({}, { serverless: false }).export_types).toMatchInlineSnapshot(`
      Object {
        "csv": Object {
          "enabled": true,
        },
        "pdf": Object {
          "enabled": true,
        },
        "png": Object {
          "enabled": true,
        },
      }
    `);
  });

  it('disables screenshot type exports in serverless', () => {
    expect(ConfigSchema.validate({}, { serverless: true }).export_types).toMatchInlineSnapshot(`
      Object {
        "csv": Object {
          "enabled": true,
        },
        "pdf": Object {
          "enabled": false,
        },
        "png": Object {
          "enabled": false,
        },
      }
    `);
  });
```